### PR TITLE
Booking view

### PR DESCRIPTION
**Pull request by goh0222**

### CODE REVIEW

File: app/assets/stylesheets/bookings/_index.scss [hunk 1]
```diff
@@ -1,27 +1,13 @@
 .booking {
-  body {
-    display: flex;
```
Comment (Rainesisu) [inline app/assets/stylesheets/bookings/_index.scss:3]: can i assume that this is being replaced by .flex-wrap-center class?

Comment (goh0222) [reply app/assets/stylesheets/bookings/_index.scss:3]: Hmmm.. not replacing anything, just that it wasn't used hence I deleted :)

File: app/javascript/bookings/index.js [hunk 2]
```diff
@@ -28,9 +36,34 @@ const updateForm = (event) => {
...
+  const priceCents = guestCount * unitPriceCents * count;
+  if (guestCount >= 1) {
+    updateGuestCounter(guestCounter, guestCount);
+    updateInput(priceCents);
+    updateCTA(priceCents);
+  } else {
+    event.preventDefault();
+  }
+};
+
```
Comment (Rainesisu) [inline app/javascript/bookings/index.js:56]: guest counter + price in cents! genius!

**Review by Rainesisu — APPROVED** [review]
saw some very good codes! amazing

**Pull request merged**